junit-platform-console/junit-platform-console.gradle

```diff
@@ -42,14 +48,16 @@ javadoc {
 jar.enabled = false
 
 shadowJar {
-	classifier = null
+	baseName = 'junit-console'
+	version = "$rootProject.version"
+	classifier = 'standalone'
 	configurations = [project.configurations.shadowed]
-	exclude 'META-INF/**'
+	mergeServiceFiles()
 	relocate 'joptsimple', 'org.junit.platform.joptsimple'
 	transform(org.junit.platform.gradle.plugins.shadow.transformers.PropertiesFileTransformer) {
 		paths = [
-			'joptsimple/ExceptionMessages.properties',
-			'joptsimple/HelpFormatterMessages.properties'
+				'joptsimple/ExceptionMessages.properties',
+				'joptsimple/HelpFormatterMessages.properties'
```

Comment:
It's not clear to me why these lines are indented by 2 relative tabs now rather than 1. @sormuras, could you explain your thought process here for me? :)